client/main.css

```diff
@@ -1288,10 +1288,16 @@ audio {
 .news-item__author {
   font-family: 'nimbus-sans', sans-serif;
   text-transform: uppercase;
-  color: #5940de;
+  color: #000;
```

Comment:
Is this necessary here? I was thinking it'd be dark text without a `color` explicitly declared.  Anyway, why black? Aren't we using a lighter #444444 or #333333 for body text?

Reply:
OK I guess black is fine?